.circleci/config.yml

```diff
@@ -2113,10 +2113,10 @@ jobs:
           command: |
             $ProgressPreference = "SilentlyContinue"
             mkdir python_installer -Force
-            if (!(Test-Path .\python_installer\python-3.8.7-amd64.exe -PathType Leaf)) {
-                wget -O .\python_installer\python-3.8.7-amd64.exe https://www.python.org/ftp/python/3.8.7/python-3.8.7-amd64.exe
+            if (!(Test-Path .\python_installer\python-3.8.10-amd64.exe -PathType Leaf)) {
+                wget -O .\python_installer\python-3.8.10-amd64.exe https://www.python.org/ftp/python/3.8.10/python-3.8.10-amd64.exe
```

Comment:
Looks like 3.8.12 is out so we may want to update to that during the next release - https://docs.python.org/release/3.8.12/whatsnew/changelog.html#python-3-8-12-final.

I don't think any of those issues affect us directly, but just to be up to date.